appIcons.js

```diff
@@ -1233,8 +1233,11 @@ function getInterestingWindows(windows, monitorIndex) {
         const showUrgent = settings.get_boolean('workspace-agnostic-urgent-windows');
         const activeWorkspace = global.workspace_manager.get_active_workspace_index();
         windows = windows.filter(function(w) {
-            return w.get_workspace().index() == activeWorkspace
-                || (showUrgent && (w.urgent || w.demandsAttention || w._manualUrgency));
+            // See GJS docs : Meta/Window
```

Comment:
This is not needed, it should be pretty known for who is into developing a shell extension :)